app/v1/models/domain/CountryCode.scala

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.domain
+
+import play.api.libs.json.Format
+import utils.enums.Enums
+
+sealed trait CountryCode
+
+object CountryCode {
+  case object `AD` extends CountryCode
```

Comment:
Honestly I think it might just be better to just return this country code as a String rather than an Enum, it means that we don't have to maintain it if the country codes change for whatever reason or we've missed anything. I appreciate that this must've been a nightmare to type out but it'll just be easier for us (and any live service teams if this repo gets handed on when it's live) if we don't have to maintain a massive list of country codes